contracts/lending/ttoken/ITToken.sol

```diff
@@ -51,7 +51,7 @@ abstract contract ITToken is ERC20Upgradeable, RolesFacet {
         returns (uint256 balance_);
```

Comment:
line 34 needs to change from `assets` to `asset`

Reply:
done